src/components/LessonGroups/index.js

```diff
@@ -55,26 +52,30 @@ export default ({instructor}) => {
       ],
       fallbackDescription: noInQueueLessonsDescriptionText,
     },
-    {
-      title: publishedTitleText,
-      description: publishedDescriptionText,
-      states: [
-        'published',
-        'flagged',
-        'revised',
-      ],
-      fallbackDescription: noPublishedLessonsDescriptionText,
-    },
-  ]
+    instructor
+      ? null
+      : {
+          title: publishedTitleText,
+          states: [
+            'published',
+            'flagged',
+            'revised',
+          ],
+          fallbackDescription: noPublishedLessonsDescriptionText,
+        },
+  ])
```

Comment:
Only show "Published" `LessonGroups` tab for non-instructor specific instances.
![image](https://cloud.githubusercontent.com/assets/5497885/24117615/84ab27ce-0d70-11e7-825b-5d98b16e4558.png)